EpiOAuth.php

```diff
@@ -384,6 +390,8 @@ public function __construct($consumerKey, $consumerSecret, $signatureMethod='HMA
     $this->consumerKey = $consumerKey;
     $this->consumerSecret = $consumerSecret;
     $this->signatureMethod = $signatureMethod;
+    $this->serverAddr = (!empty($_SERVER['SERVER_ADDR']) && $_SERVER['SERVER_ADDR'] != '127.0.0.1')
```

Comment:
It just to save old functionality. If you don't set up the serverAddr value,  everything will be work as before. But if you set up the value to false, the "CURLOPT_INTERFACE" will be skipped, if you set up certain value, it will be used for "CURLOPT_INTERFACE".